eecms-docs.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This file needs noy be in Git

Reply:
Dammed! It should be on `.gitignore`. Dammed.

Reply:
Oh, you've gitignored it on other repo :)